Makefile

```diff
@@ -11,16 +40,9 @@ add: pyproject.toml
 uninstall: pyproject.toml
 	poetry remove $(package)
 
-# Migrations
-migrations:
-	python manage.py makemigrations
-
-migrate:
-	python manage.py migrate
-
-# Tests
-tests:
-	python manage.py test
+# Admin
+superuser:
+	python manage.py createsuperuser
 
 # Docker
```

Comment:
you can remove all the docker commands under this comment as we're not using them